src/App.jsx

```diff
@@ -35,15 +51,21 @@ useEffect(()=> {
         price="Coming soon"
       />
 
```

Comment:
This part can be removed since we now have the proper places.json file.

Reply:
done